### PR TITLE
notebook

### PR DESCRIPTION
**Pull request by Ankit-Bhagat851**

### CODE REVIEW

**Comment by Ankit-Bhagat851** [comment]
jupyter notebook created, python script created, local development set, networkx installed

File: 2023-round-1/Ankit-Bhagat851/NetwrokX.ipynb
Comment (paulitapb) [inline 2023-round-1/Ankit-Bhagat851/NetwrokX.ipynb:1]: The plots you present here are not histograms. There's a key difference between a barplot and a histogram. In a histogram, you see the frequency distribution of a variable (in this case the variable is the degree centrality). 

Also, could you rename this file `nx_notebook.ipynb`?

Comment (Ankit-Bhagat851) [reply 2023-round-1/Ankit-Bhagat851/NetwrokX.ipynb:1]: I have done the changes. Have  a look

**Comment by Ankit-Bhagat851** [comment]
Should I make a new pull request?

**Comment by MridulS** [comment]
Closing this as contribution period is over. Thanks!

**Pull request closed without merging**